CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.1.0)
 
 project(sway C)
 
-set(CMAKE_C_FLAGS "-g")
+set(CMAKE_C_FLAGS "-g -lm")
```

Comment:
This is required by `math.h`, if this is not the right way to do this let me know.